### ISSUE TITLE
starting container process caused 

### ISSUE THREAD

**Issue by prasenforu** [concourse/concourse#2359]
First Issue:

Based on tutorial created very small pipeline in concourse.

```
resources:
- name: git-dockerapp
  type: git
  source:
    uri: https://github.com/prasenforu/dockerapp.git
    branch: master

- name: push-harbor-image
  type: docker-image
  source:
    username: "uuuuu"
    password: "ppppp"
    repository: "harbor:443/ocp-dev/hello-world"
    tag: "latest"

jobs:
- name: build-docker-image
  public: true
  serial: true
  plan:
  - get: git-dockerapp
    trigger: true
  - put: push-harbor-image
    params:
      build: git-dockerapp
```
then run following command 

```
fly -t hello sp -p hello-world -c pipeline.yml
```
Then I can see web-ui

![image](https://user-images.githubusercontent.com/19322497/42431764-f5fac8ba-8364-11e8-8db6-0a6d89d69781.png)

After clicking "git-dockerapp" getting following error.

```
runc run: exit status 1: container_linux.go:348: starting container process caused "process_linux.go:301: running exec setns process for init caused \"exit status 40\""
```
![image](https://user-images.githubusercontent.com/19322497/42431792-1a7233f4-8365-11e8-9ffe-6f61ad181133.png)

* fly version: 3.14.1
* Deployment type (BOSH/Docker/binary): Docker
* Infrastructure/IaaS: AWS
* OS : CentOS 
* Browser (if applicable):
* Did this used to work?

**Comment by electricsam** [comment]
I'm having the same issue on RHEL 7.4

**Comment by electricsam** [comment]
I fixed this issue by following the instructions here: https://success.docker.com/article/user-namespace-runtime-error

I did not update the daemon.json as described though.  That caused more issues.

It is all a mute point for RHEL 7.4 though as I got another error 

`runc start: exit status 1: unknown capability "CAP_AUDIT_READ"`

This one was a show stopper for Concourse.  It turns out RHEL does not have a new enough kernel:  https://github.com/concourse/concourse/issues/503
